docs/docs/understanding/workflows/branches_and_loops.md

```diff
@@ -0,0 +1,89 @@
+# Branches and loops
+
+A key feature of Workflows is their enablement of branching and looping logic, more simply and flexibly than graph-based approaches.
```

Comment:
"More simply and flexibly" -- kind of awkward. Maybe "with more simplicity and flexibility compared to graph-based approaches"